ci.nix

```diff
@@ -132,17 +132,19 @@ rec {
     (mapAttrs (pkgName: pkg: optional (pkg ? library) pkg.library.haddock) packages));
 
   # run some executables to produce contract documents
-  contracts-doc = { release, commitSha ? null, commitDate ? null }@releaseArgs: pkgs.runCommand "contracts-doc" {
+  contract-doc = { release, commitSha ? null, commitDate ? null }@releaseArgs: pkgs.runCommand "contract-doc" {
     buildInputs = [
-      (hs-pkgs releaseArgs).autodoc-sandbox.components.exes.autodoc-sandbox-registry
+      (hs-pkgs releaseArgs).homebase-lite.components.exes.homebase-lite
     ];
   } ''
     mkdir $out
-    cd $out
-    mkdir autodoc
-    autodoc-sandbox-registry document --name AutodocSandbox --output \
-      autodoc/AutodocSandboxContract.md
+    homebase-lite document --name Homebase-Lite --output \
+      $out/documentation.md
```

Comment:
I believe that the `--name` option is no longer needed here and can be removed (which seem to be why CI is failing ATM).